### PR TITLE
Quick sort in Rust

### LINKED ISSUE #2131: Quick Sort
- [x] C
- [x] C++
- [x] C# - @Yashank18
- [ ] CoffeeScript - @prashansag62
- [x] Dart
- [x] Go
- [x] Java
- [x] JS
- [x] Kotlin - @97shivank
- [x] Php - @goyalmuskan
- [x] Python
- [x] Readme
- [x] Ruby
- [ ] Rust - @m-code12
- [ ] TypeScript - @dj373

### PR DESCRIPTION
**Pull request by m-code12**
Fixes #2131 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added Quick Sort in Rust

#### Languages Used:
- Rust

#### Files Added:
- Quick_Sort.rs

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Quick_Sort/Quick_Sort.rs [hunk 1]
```diff
@@ -0,0 +1,66 @@
+use std::io;
+use std::str::FromStr;
+
+fn main() {
+    println!("Enter the numbers to be sorted (separated by space) : ");
+    let mut array: Vec<u32> = read_values :: <u32>().unwrap();
+    quicksort(&mut array, 0, array.len()-1);
```
Comment (singh-shreya6) [inline Quick_Sort/Quick_Sort.rs:7]: Space around -

**Comment by singh-shreya6** [comment]
Squashing not done properly.

**Comment by m-code12** [comment]
I am getting errors, but I  am trying

**Comment by m-code12** [comment]
sorry I did not see your comment

**Comment by singh-shreya6** [comment]
It is okay, since you have done it. I had not merged it yet.

**Comment by m-code12** [comment]
i have squashed commits

**Pull request merged**